ops/build_and_package.sh

```diff
@@ -33,3 +33,31 @@ go build
 echo "Packaging bcdaworker binary into RPM..."
 fpm -v $VERSION -s dir -t rpm -n bcdaworker bcdaworker=/usr/local/bin/bcdaworker
 
+#Sign RPMs
+GPG_RPM_USER="2AFAC6BABEFC07B6"
+GPG_RPM_EMAIL="bcda-ops.group@adhocteam.us"
+GPG_PUB_KEY_FILE="../ops/RPM-GPG-KEY-bcda"
+GPG_SEC_KEY_FILE="../ops/RPM-GPG-KEY-SECRET-bcda"
+WORKER_RPM="bcdaworker-*.rpm"
```

Comment:
Consider setting all constants at the top of the script for readability.

Reply:
Please add a check at the top of the script which fails execution if any of the env vars are not set.